contents/handbook/people/onboarding.md

```diff
@@ -4,566 +4,90 @@ sidebar: Handbook
-</li>
-</ul>
-   </td>
-  </tr>
-  <tr>
-   <td>I-9 Check
-   </td>
-   <td>US
-   </td>
-   <td>Eltje
-   </td>
-   <td>
-<ul>
-
-<li><code>[ ]</code>
-</li>
-</ul>
-   </td>
-  </tr>
-  <tr>
-   <td>Team members should add themselves to the <a href="https://calendar.google.com/calendar/?cid=Y19tczllaWN1Ym92ZGgxYWhzNmtoY2xpNTQ3b0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t">customer interviews calendar</a>.
-   </td>
-   <td>All
-   </td>
-   <td>Team member
-   </td>
-   <td>
-<ul>
-
-<li><code>[ ]</code>
-</li>
-</ul>
-   </td>
-  </tr>
-  <tr>
-   <td>Share user interview notes with them, found in <a href="https://docs.google.com/document/d/1762fbEbFOVZUr24jQ3pFFj91ViY72TWrTgD-JxRJ5Tc/edit">this doc</a>. If the person is particularly interested in more historical context, here are the notes from <a href="https://docs.google.com/document/d/1gJlsUDrlW7ur8zT5scqRvXZhapm_0JdvKGiw68Iyx9E/edit">Q4-2020</a>, and <a href="https://docs.google.com/document/d/1vrwn-owF320otkm3oODCFjvqj7gptF6QaFFO6v-_RhY/edit">Q3-2020</a>.
-   </td>
-   <td>All
-   </td>
-   <td>Manager
-   </td>
-   <td>
-<ul>
-
-<li><code>[ ]</code>
-</li>
-</ul>
-   </td>
-  </tr>
-    <tr>
-   <td>Identify user as team member in <a href="https://app.orbit.love/posthog" target="_blank">Orbit</a>
-   </td>
-   <td>All
-   </td>
-   <td>Anyone
-   </td>
-   <td>
-<ul>
-
-<li><code>[ ]</code>
-</li>
-</ul>
-   </td>
-  </tr>
-</table>
-
-
-> Not a hard requirement by any means, but we highly recommended that you join a feedback call and/or product demo in your first weeks. It provides great context on our users.
+*   [ ] Book a weekly 1:1 with the team member - _Manager_
+*   [ ] Send them these instructions on adding the [team time off cal](https://intercom.help/charliehr/en/articles/839648-importing-your-time-off-calendar-to-google-calendar) to their Gcal - _Eltje_
+*   [ ] For the first week or so, book extra sessions as appropriate to provide extra help - _Manager_
+*   [ ] Book Welcome/Intro session - _Eltje_ 
+*   [ ] (_UK only)_ Schedule a [right to work](https://www.gov.uk/guidance/coronavirus-covid-19-right-to-work-checks) check - _Eltje_
+*   [ ] (U_S only_) I-9 Check - _Eltje_ 
+*   [ ] Team members should add themselves to the [customer interviews calendar](https://calendar.google.com/calendar/?cid=Y19tczllaWN1Ym92ZGgxYWhzNmtoY2xpNTQ3b0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) - _Team member_
+*   [ ] Identify user as team member in [Orbit](https://app.orbit.love/login) - _Anyone_
+*   [ ] Share user interview notes with them, found in [this doc](https://docs.google.com/document/d/1762fbEbFOVZUr24jQ3pFFj91ViY72TWrTgD-JxRJ5Tc/edit). If the person is particularly interested in more historical context, here are the notes from [Q4-2020](https://docs.google.com/document/d/1gJlsUDrlW7ur8zT5scqRvXZhapm_0JdvKGiw68Iyx9E/edit), and [Q3-2020](https://docs.google.com/document/d/1vrwn-owF320otkm3oODCFjvqj7gptF6QaFFO6v-_RhY/edit) - _Manager_
+
+## New joiner checklist - to be done during their first week
```

Comment:
@shehuphd I can't find the new joiner announcement you were thinking about. Could you add it here?

Reply:
Added